github/ci/prow/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-postsubmits.yaml

```diff
@@ -16,11 +20,17 @@ postsubmits:
           zone: ci
         containers:
           - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+            env:
+            - name: GIT_ASKPASS
+              value: "../project-infra/hack/git-askpass.sh"
+            - name: GITHUB_USER
+              value: kubevirt-bot
+            command:
             command:
               - "/usr/local/bin/runner.sh"
               - "/bin/bash"
               - "-c"
-              - "cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io && ./automation/publish.sh"
+              - "cat $QUAY_PASSWORD | docker login --username $(cat $QUAY_USER) --password-stdin=true quay.io ./automation/publish.sh"
```

Comment:
You dropped && and now the command is invalid IIUIC

Reply:
Done